orchestrator-salt/src/main/resources/salt/salt/disks/service/scripts/mount-instance-storage.j2

```diff
@@ -184,10 +187,33 @@ find_format_and_mount_ephemeral_storage_aws() {
     return $((return_code))
 }
 
+semaphore_file_exists() {
```

Comment:
Is this function used in the script? Or is it used from somewhere else?

Reply:
I forgot to delete this one.